### PR TITLE
add metadata

### PR DESCRIPTION
**Pull request by noraj**
will add links to rubygems & rubytoolbox

### CODE REVIEW

File: gli.gemspec [hunk 1]
```diff
@@ -12,6 +12,15 @@ spec = Gem::Specification.new do |s|
   s.summary = "Build command-suite CLI apps that are awesome."
   s.description = "Build command-suite CLI apps that are awesome.  Bootstrap your app, add commands, options and documentation while maintaining a well-tested idiomatic command-line app"
 
+  s.metadata = {
+    'yard.run'          => 'yard',
```
Comment (davetron5000) [inline gli.gemspec:16]: What's this one do?  Is it just "what command do I run to do yard doc generation?"

Comment (noraj) [reply gli.gemspec:16]: > You can also add the following to your .gemspec to have YARD document your gem
on install:
>
>    spec.metadata["yard.run"] = "yri" # use "yard" to build full HTML docs.

or

https://github.com/lsegal/yard/issues/1187

**Comment by davetron5000** [comment]
Closing due to being very old.

**Comment by noraj** [comment]
> Closing due to being very old.

Why closing it? It's still relevant. No metadata has been added to the gemspec in the meantime. I also answered to the question you asked in the review.

**Comment by davetron5000** [comment]
Yeah, agreed it's still relevant. I still don't understand what the yard key is for since this project doesn't use yard.  I took your PR, then changed it to remove the yard key, correct one of the URLs and add a wiki link: See #328 

Thanks for the nudge!

**Comment by davetron5000** [comment]
Released in 2.21.4

**Pull request closed without merging**